### PR TITLE
daemon-hentai.com

### PR DESCRIPTION
**Pull request by piquark6046**
### To avoid invalid pull requests, please check and confirm following terms

- [X] This is not an ad/bug report;
- [X] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [X] I have performed a self-review of my own changes;
- [X] My changes do not break web sites, apps and files structure.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [ ] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [X] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [ ] Filters maintenance.

### Enter the issue address

 - https://github.com/AdguardTeam/AdguardFilters/issues/134680

### Add your comment and screenshots

None

### Terms

- [X] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: BaseFilter/sections/antiadblock.txt [hunk 1]
```diff
@@ -165,7 +165,7 @@ articlix.com,fcc.lc,fantastic-film.online,joinfaucet.com#@#.TextAd
 ! SECTION: BlockAdBlock
 !START: BlockAdBlock
 ! Old and 4.2b - #%#//scriptlet("prevent-bab")
```
Comment (zloyden) [inline BaseFilter/sections/antiadblock.txt:167]: 1. The title of the pull request (you don't have to specify it there) and the commit have wrong task number.
2. Do you need `#%#//scriptlet("prevent-bab")`? It works fine without it.

Comment (zloyden) [reply BaseFilter/sections/antiadblock.txt:167]: Btw, it does not look like a `bab` detection. More like `antiblock.org g207` for which we have rules.

Comment (zloyden) [reply BaseFilter/sections/antiadblock.txt:167]: <details><summary>Screenshot:</summary>

image

</details><br/>
You have reverted the wrong title. It is not needed.

Comment (piquark6046) [reply BaseFilter/sections/antiadblock.txt:167]: > The title of the pull request (you don't have to specify it there) and the commit have wrong task number.

Okay.

> Btw, it does not look like a bab detection. More like antiblock.org g207 for which we have rules.

I pushed a commit related to g207 anti-adblock at https://github.com/AdguardTeam/AdguardFilters/pull/134956/commits/4031de410d40351b9be2febd152045bb89cdba5a. But, I cannot see blockadblock script on the website too unlike when i check at first time.

**Review by Alex-302 — APPROVED** [review]

File: BaseFilter/sections/antiadblock.txt (hunk 1 above)
Comment (zloyden) [reply BaseFilter/sections/antiadblock.txt]: Don't worry, got it :)

**Pull request merged**